build/patches/Disable-NTP-remote-suggestions-by-default.patch

```diff
@@ -2,42 +2,12 @@ From: csagan5 <32685696+csagan5@users.noreply.github.com>
 Date: Thu, 12 Oct 2017 08:15:17 +0200
 Subject: Disable NTP remote suggestions by default
 
+License: GPL-3.0-only - https://spdx.org/licenses/GPL-3.0-only.html
 ---
- .../chrome/browser/feed/FeedSurfaceMediator.java    | 13 ++++++++++---
- components/feed/core/shared_prefs/pref_names.cc     |  4 ++--
- components/ntp_snippets/features.cc                 |  4 ++--
- 3 files changed, 14 insertions(+), 7 deletions(-)
+ components/feed/core/shared_prefs/pref_names.cc | 4 ++--
+ components/ntp_snippets/features.cc             | 4 ++--
+ 2 files changed, 4 insertions(+), 4 deletions(-)
 
-diff --git a/chrome/android/feed/core/java/src/org/chromium/chrome/browser/feed/FeedSurfaceMediator.java b/chrome/android/feed/core/java/src/org/chromium/chrome/browser/feed/FeedSurfaceMediator.java
---- a/chrome/android/feed/core/java/src/org/chromium/chrome/browser/feed/FeedSurfaceMediator.java
-+++ b/chrome/android/feed/core/java/src/org/chromium/chrome/browser/feed/FeedSurfaceMediator.java
-@@ -294,9 +294,13 @@ public class FeedSurfaceMediator
-     /** Update the content based on supervised user or enterprise policy. */
-     void updateContent() {
-         mFeedEnabled = FeedFeatures.isFeedEnabled();
--        if (mFeedEnabled == !mTabToStreamMap.isEmpty()) {
--            return;
--        }
-+        // Bromite needs the next lines removed because mFeedEnabled
```

Comment:
it seems to me that this whole part is missing

Reply:
The feed Java files are not built, they are excluded with modifications to `BUILD.gn` files.

Reply:
> The feed Java files are not built, they are excluded with modifications to BUILD.gn files

yes, you are right, I modified them before removing it from the gn's